product-service/src/getAllProducts.js

```diff
@@ -0,0 +1,40 @@
+const { Client } = require("pg");
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbOptions = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false, // to avoid warring in this example
+  },
+  connectionTimeoutMillis: 5000, // time in millisecond for termination of the database query
+};
+
+export const getAllProducts = async (event) => {
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  console.log("Lambda invocation with event: ", event);
+  try {
+    const { rows: products } = await client.query(
+      "select * from products join stocks on stocks.product_id = products.id"
```

Comment:
Currenly, endpoint returns this:
```
[
    {
        "id": "357e6028-cbf2-4647-b085-2ca144dfad77",
        "title": "Item #1 from PG",
        "description": "First item added to a database",
        "price": 100,
        "product_id": "d22f957c-17a1-43bd-8b47-82bede3ce204",
        "count": 2
    },
...
]
```
As you see, there are two ids, both of them should be product's ids, but they are different, because one of them a stock id. Please, fix this query. And even better to remove unnecessary id, because we need only one id for a product in response.